### PR TITLE
DOCSP-30407-compass-release-notes

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
## DESCRIPTION

1.38 Compass release notes.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-30407-compass-release-notes/release-notes/#compass-1.38.0

## JIRA

https://jira.mongodb.org/browse/DOCSP-30407

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64937f65423952aeb92f3360

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,58 @@ Release Notes
...
+
+*Released June 21, 2023*
+
+New Features:
+
+- OIDC authentication (:issue:`COMPASS-6803`).
+- Stage wizard (:issue:`COMPASS-6814`).
+- Add visual tree and update summary for the aggregation explain plan
+  (:issue:`COMPASS-6821` and :issue:`COMPASS-6888`).
+- Add types docs link to import preview (:issue:`COMPASS-6632`).
```
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:24]: (question)
This seems fairly internal and probably safe to delete. Would you please check during the external review?

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,58 @@ Release Notes
...
+- Open a collection in a new tab shortcut.
+- Add performance tab indicator to state that information about certain
+  collections is missing (:issue:`COMPASS-6593`).
+- Add zoom control to the modal tree view (:issue:`COMPASS-6843`).
```
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:28]: (question)
This seems fairly internal. Does it make sense to delete?

File: source/release-notes.txt [hunk 3]
```diff
@@ -10,6 +10,58 @@ Release Notes
...
+- Add Atlas error message when connection fails because of IP access
+  issue (:issue:`COMPASS-6842`).
+- Show insight when query is unindexed (:issue:`COMPASS-6832`).
+- Show insight when number of collections is too high
+  (:issue:`COMPASS-6835`).
+- Add unindexed aggregation insight (:issue:`COMPASS-6833`).
+- Guide Cue component (:issue:`COMPASS-6334`).
```
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:35]: (suggestion) 
Can you turn this one into a short sentence?

File: source/release-notes.txt [hunk 4]
```diff
@@ -10,6 +10,58 @@ Release Notes
...
+- Let ``enableShell`` setting control the runtime start and stop.
+- Track ``has_sort`` in telemetry.
+- Various jQuery fixes (:issue:`COMPASS-6885`, :issue:`COMPASS-6884`,
+  :issue:`COMPASS-6883`, and :issue:`COMPASS-6882`).
+- Replace ``got`` with ``fetch`` in redirect (:issue:`COMPASS-6881`).
+- Fix redirects (:issue:`COMPASS-6880` and :issue:`COMPASS-6879`).
+- Show 1 as the page number for collections where no entries are
+  present.
+- Select combo box option ``onBlur`` issue (:issue:`COMPASS-6511`).
+- Use ``mongodb-cloud-info`` v2 for ipv6 support in cloud metrics
```
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:55]: (terminology)
IPv6

**Review by sarah-olson-mongodb — APPROVED** [review]
Nice job @jason-price-mongodb. There are a few issues that look to be really internal. It's worth asking about that in the external review. Can any of these items be removed? 

One minor terminology nit. 
And throughout the changes, can you do the very lightest of a copy edit pass. I no we don't do tons of editorial work here, but it's worth making sure that we have solid grammar.

**Review by jason-price-mongodb — COMMENTED** [review]
@sarah-olson-mongodb please re-review. Thanks.

File: source/release-notes.txt (hunk 1 above)
Comment (Anemy) [reply source/release-notes.txt:25]: Yup!

File: source/release-notes.txt (hunk 2 above)
Comment (Anemy) [reply source/release-notes.txt:29]: Yup!

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks Jason. Appreciate the additional updates and fixes.

File: source/release-notes.txt (hunk 2 above)
Comment (jason-price-mongodb) [reply source/release-notes.txt]: Done.

File: source/release-notes.txt (hunk 1 above)
Comment (jason-price-mongodb) [reply source/release-notes.txt]: Done.

**Review by Anemy — APPROVED** [review]

**Review by jason-price-mongodb — COMMENTED** [review]
Done.

**Pull request merged**